package.json

```diff
@@ -85,7 +85,8 @@
     "webpack": "4.x.x"
   },
   "buildDependencies": {
-    "babel": "^5.8.23"
+    "babel": "^5.8.23",
+    "uglifyjs-webpack-plugin": "^2.1.1"
```

Comment:
this should not be needed, did you add this because you could not build? If so, it might be fixed with #436